pkg/logger/logger.go

```diff
@@ -16,10 +17,10 @@ var (
 
 // Log level constants.
 const (
-	ErrorLevel = iota + 1
-	WarningLevel
-	InfoLevel
-	DebugLevel
+	ERROR_LEVEL = iota + 1
```

Comment:
The chosen linter does not like all caps CONST's 

Does the team prefer the original CamelCase? If so I will change it back.

If the team likes what I have done here I can look at modifying the linter, 
Let me know what you think :smile:

Reply:
we've mostly stuck to camelcase, so might be best to stick to that

Reply:
Thanks for the feedback, I have reverted this change :smile: